packages/dds/counter/README.md

```diff
@@ -153,24 +171,42 @@ This causes the `incremented` event to be sent to all of the clients who have th
 
 Since `updateCounterValueLabel` is listening for all `incremented` events, the view will always refresh with the appropriate updated value any time a collaborator increments or decrements the counter value.
 
-<!-- AUTO-GENERATED-CONTENT:START (README_API_DOCS_SECTION:includeHeading=TRUE) -->
+<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README:scripts=FALSE&installation=FALSE&importInstructions=FALSE) -->
```

Comment:
Opted to use the "whole-readme" template (excluding the 2 sections placed towards the top of the README) to simplify things and to ensure future contents appear automatically as we add them.

Longer term, we may want to add package README "header" and "footer" pragmas to simplify this pattern, which seems to have become the standard across the repo.